### PR TITLE
Add compatibility with 10.11

### PR DESCRIPTION
**Pull request by tectiv3**
## What this PR does ?
goapp now can build and bundle apps for macOS 10.11
## Fixes
10.11 compatibility

### CODE REVIEW

File: cmd/goapp/mac.go [hunk 1]
```diff
@@ -213,7 +213,7 @@ func fillBundle(b driver.Bundle, root string) driver.Bundle {
 	}
 
 	if len(b.DeploymentTarget) == 0 {
-		b.DeploymentTarget = "10.13"
+		b.DeploymentTarget = "10.11"
```
Comment (maxence-charriere) [inline cmd/goapp/mac.go]: I would like to keep the latest version as a default value.

**Comment by tectiv3** [comment]
Something like this?

File: drivers/mac/image.h [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 #import <Cocoa/Cocoa.h>
 
+#if MAC_OS_X_VERSION_MAX_ALLOWED < 101200
```
Comment (maxence-charriere) [inline drivers/mac/image.h]: Get rid of these #if and put it just once in retro.h

**Comment by tectiv3** [comment]
done :)

**Comment by maxence-charriere** [comment]
Thank you for this :)

Sent from my iPhone

On Jul 8, 2018, at 2:02 AM, tectiv3 <notifications@github.com<mailto:notifications@github.com>> wrote:


done :)

**Pull request merged**